Resources/Prototypes/_StarLight/Voice/speech_emotes.yml

```
@@ -356,4 +356,37 @@
   - rattles
   - rattles.
   - rattles!
-  - rattles?
\ No newline at end of file
+  - rattles?
+
+# Starlight: Port wing flap sound from Goob
+- type: emote
+  id: FlapWings
+  name: chat-emote-name-flap-wings
+  category: Vocal
+  available: false
+  icon:
+    sprite: _Starlight/Actions/moth.rsi
+    state: flap_wings
```

```suggestion
  icon:
    sprite: Clothing/Neck/Cloaks/moth.rsi
    state: icon
```
You could use the current moth cloak icon instead of re-using the same icon (save space!)